_docs/2.0.0/usage/config.md

```diff
@@ -30,5 +30,12 @@ spark-submit --conf "spark.driver.extraJavaOptions= -Dkey1=value1 -Dkey2=value2"
 
 <!-- specific sections on Standardization & Conformance options may follow in the future -->
 
+Note, that
+  - `conformance.autoclean.standardized.hdfs.folder` when set to true and the job is writing to S3, there will be a leftover empty file like `conformance-output_$folder$` after the autoclean.
```

Comment:
I would like to have this note linked from the `conformance.autoclean.standardized.hdfs.folder`, similarly as you can see in https://absaoss.github.io/enceladus/2021/02/12/release-2.18.1.html

Reply:
```suggestion
  - `conformance.autoclean.standardized.hdfs.folder` when set to true and the job is writing to S3, there could be a leftover empty file like `conformance-output_$folder$` after the autoclean.
```
The file is put there only in case the folder would be all empty. If there are other folders, it won't be created.